content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -11,6 +11,12 @@ resource_types:
       repository: governmentpaas/s3-resource
       tag: latest
 resources:
+  - name: slack-webhook
+    type: slack-alert
+    check_every: never
+    source:
+      url: ((slack-url))
+      disabled: false
```

Comment:
Elsewhere we wrap this in NON-DEV only. But I don't think there's really any need. If we include this in dev and a failure happens, you'll just see an error about the url being bad.

The case for not using this in dev might be if we sent alerts on success, because then it would fail an otherwise succeeding pipeline.